src/publisher.js

```diff
@@ -130,21 +147,59 @@ Publisher.prototype.publish = function () {
 
 					http(config, function (error, response) {
 						if (!error && response.statusCode == 200) {
```

Comment:
This might end up being a conflict with #16, but should be easy enough to resolve.